packages/wxt/src/core/resolve-config.ts

```diff
@@ -426,17 +424,21 @@ async function getUnimportEslintOptions(
  * Returns the path to `node_modules/wxt`.
  */
 function resolveWxtModuleDir() {
-  // TODO: Use this once we're fully running in ESM, see https://github.com/wxt-dev/wxt/issues/277
-  // const url = import.meta.resolve('wxt', import.meta.url);
-  // resolve() returns the "wxt/dist/index.mjs" file, not the package's root
-  // directory, which we want to return from this function.
-  // return path.resolve(fileURLToPath(url), '../..');
-
-  const nodeRequire = globalThis.require ?? createRequire(import.meta.url);
-
-  // resolve() returns the "wxt/dist/index.mjs" file, not the package's root
+  // TODO: Drop the __filename expression once we're fully running in ESM
+  // (see https://github.com/wxt-dev/wxt/issues/277)
+  const importer =
+    typeof __filename === 'string'
+      ? pathToFileURL(__filename).href
+      : import.meta.url;
+
+  // TODO: Switch to import.meta.resolve() once the parent argument is unflagged
+  // (e.g. --experimental-import-meta-resolve) and all Node.js versions we support
+  // have it.
+  const url = esmResolve('wxt', importer);
```

Comment:
I could've stuck with `createRequire` here, since we're only using a parent directory of the resolved entry point, so it doesn't matter if a CJS module is returned, but I decided it was cleaner to use `esmResolve` since we have it in scope. LMK if you disagree.